docs/configuration.rst

```diff
@@ -14,11 +14,11 @@ To use a custom configuration file, use the ``-c`` option:
 If ``-c`` is not provided, yamllint will look for a configuration file in the
 following locations (by order of preference):
 
-- ``.yamllint``, ``.yamllint.yaml`` or ``.yamllint.yml`` in the current working
-  directory
-- the file referenced by ``$YAMLLINT_CONFIG_FILE``, if set
-- ``$XDG_CONFIG_HOME/yamllint/config``
-- ``~/.config/yamllint/config``
+- A file named ``.yamllint``, ``.yamllint.yaml``, or ``.yamllint.yml`` in the
+  current working directory
+- The filename referenced by ``$YAMLLINT_CONFIG_FILE``, if set
+- The file ``$XDG_CONFIG_HOME/yamllint/config`` or
+  ``~/.config/yamllint/config``, if present
```

Comment:
I wonder if this would be more consistent with the above
```suggestion
- A filename referenced by ``$YAMLLINT_CONFIG_FILE``, if set
- A file named ``$XDG_CONFIG_HOME/yamllint/config`` or
  ``~/.config/yamllint/config``, if present
```

I'm unsure.

Reply:
Yes it would probably be :+1: